oauth2client/service_account.py

```diff
@@ -581,24 +581,6 @@ def __init__(self,
             revoke_uri=revoke_uri,
             **additional_claims)
 
-    def authorize(self, http):
```

Comment:
This can be deleted since the base already does it and the new `inject_` holds everything?